cmd/get/execution.go

```diff
@@ -22,24 +22,27 @@ Retrieves all the executions within project and domain.(execution,executions can
  bin/flytectl get execution -p flytesnacks -d development
 
 Retrieves execution by name within project and domain.
-
 ::
 
  bin/flytectl get execution -p flytesnacks -d development oeh94k9r2r
 
-Retrieves execution by filters
+Retrieves all the tasks with filters.
 ::
 
- Not yet implemented
+ bin/flytectl get execution -p flytesnacks -d development oeh94k9r2r --filters="eq(execution.name, "oeh94k9r2r")" 
```

Comment:
can you list all variants of the filters - `ne, eq, ..`
Also are combination filters supported - for e.g. 
--filters="eq(..)" --filters "ne(...)"